examples/web-wagmi/src/revenue/components/PublicationRevenueCard.tsx

```diff
@@ -19,9 +19,9 @@ export function PublicationRevenueCard({ publicationRevenue }: RevenueCardProps)
         <p>{publicationRevenue.publication.metadata.content}</p>
       </div>
 
-      <p>{`Currency: ${publicationRevenue.revenue.total.asset.name} `}</p>
-      <p>{`Symbol: ${publicationRevenue.revenue.total.asset.symbol}`}</p>
-      <p>{`Amount: ${publicationRevenue.revenue.total.value}`}</p>
+      <p>{`Currency: ${publicationRevenue.revenue.totalAmount.asset.name} `}</p>
+      <p>{`Symbol: ${publicationRevenue.revenue.totalAmount.asset.symbol}`}</p>
+      <p>{`Amount: ${publicationRevenue.revenue.totalAmount.toFixed(2)}`}</p>
```

Comment:
This is already fixed in other PR but i added it here since i needed it for the CI to pass